chia/wallet/wallet_coin_store.py

```diff
@@ -51,8 +51,18 @@ async def create(cls, wrapper: DBWrapper2):
 
             await conn.execute("CREATE INDEX IF NOT EXISTS wallet_id on coin_record(wallet_id)")
 
+            await conn.execute("CREATE INDEX IF NOT EXISTS coin_amount on coin_record(amount)")
```

Comment:
could this index slow down the DB? or is it just more space on disk? is it a large amount of space for mainnet?

Reply:
it's more space and it slows down inserts

Reply:
this only scales by the size of the wallet though, not the size of the full blockchain. But it's not ideal, a better solution would be to slightly alter the definition of the tx limit to something that's easier to query or maintain in memory.

Reply:
oh right I forgot this was wallet. not a big deal then